qemu/tests/cfg/time_conv.cfg

```diff
@@ -0,0 +1,17 @@
+- time_conv:
+    virt_test_type = qemu
+    type = time_conv
+    required_qemu = [5.1.0-2, )
+    start_vm = no
```

Comment:
No vm for this case,  suggest add `take_regular_screendumps = no`.

Reply:
> No vm for this case, suggest add `take_regular_screendumps = no`.

I just checked log, no screendump log.